src/metabase/driver/generic_sql.clj

```diff
@@ -453,6 +458,7 @@
   "Default implementations of methods in `IDriver` for SQL drivers."
   []
   (require 'metabase.driver.generic-sql.query-processor)
+
```

Comment:
In general we try to follow the Clojure Community Style Guide pretty strictly. So here I would vote against adding a blank line in this function body. See https://github.com/bbatsov/clojure-style-guide#no-blank-lines-within-def-forms